pysparkling/context.py

```diff
@@ -189,6 +189,10 @@ def newRddId(self):
         Context.__last_rdd_id += 1
         return Context.__last_rdd_id
 
+    @property
+    def defaultParallelism(self):
+        return 1
```

Comment:
This method is later used by Spark SQL to determine in some cases how many partitions a DataFrame should have

In case of multiprocessing it would be nice to return the size of the Context's pool, but I didn't find a clear method in ProcessPoolExecutor/multiprocessing.Pool/ThreadPoolExecutor to retrieve this information